docs/tactics.md

```diff
@@ -973,3 +975,25 @@ by vampire
 5. $false [resolution 4,3]
 "
 ```
+`vampire` only proves valid formulas of FOL, which means that all the necessary hypotheses should be included in the goal as antecedents.
+```
+example (p q : Prop) (h1 : p) (h2 : q) : p ∧ q :=
+by vampire -- fails
+```
+You can include the names of relevant hypotheses in order to revert them and call `vampire` in one step.
+```
+example (p q : Prop) (h1 : p) (h2 : q) : p ∧ q :=
+by vampire h1 h2
+```
+Use the `all` keyword to revert all hypotheses of type `Prop`. Hypothesis reversion also works in offline mode.
```

Comment:
Can't the tactic just inspect the goal, and determine if any goals need to be reverted?

Reply:
Proof search is most efficient when the goal is kept as small as possible by only reverting relevant hypotheses, so I think it's a good idea to make the tactic conservative by default.